.github/workflows/codegraph.yml

```
@@ -12,15 +12,16 @@ jobs:
       - name: Call Codegraph document generation API
         shell: bash
         env:
-          CODEGRAPH_API_TOKEN: ${{ secrets.CODEGRAPH_API_TOKEN }}
+          CODEGRAPH_API_TOKEN $ secrets.CODEGRAPH_API_TOKEN 
```

actionlint (1.7.4)

expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)